pruning/src/lib.rs

```diff
@@ -23,9 +23,11 @@ use std::cmp::Ordering;
 use thiserror::Error;
 
 pub const CRYPTONOTE_MAX_BLOCK_NUMBER: u64 = 500000000;
```

Comment:
Is this the max block height?

Reply:
yes, I'll rename this to `CRYPTONOTE_MAX_BLOCK_HEIGHT` to make it more clear